sdk/python/kfp/compiler/_component_builder.py

```diff
@@ -328,11 +340,15 @@ def _generate_entrypoint(self, component_func, python_version='python3'):
     for input_arg in input_args:
       func_signature += input_arg + ','
     func_signature += '_output_file):'
+    if output_is_named_tuple:
+      func_signature = func_signature.replace('file', 'files')
```

Comment:
This might be fragile.
Let's do this:
```python
if output_is_named_tuple:
  func_signature += '_output_files'
else:
  func_signature += '_output_file'
func_signature += '):'
```

Reply:
Sounds good--updated!